sdk/alertsmanagement/Azure.ResourceManager.AlertsManagement/api/Azure.ResourceManager.AlertsManagement.netstandard2.0.cs

```diff
@@ -0,0 +1,644 @@
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.AlertsManagement.Models.AlertsSummaryGroupByField left, Azure.ResourceManager.AlertsManagement.Models.AlertsSummaryGroupByField right) { throw null; }
+        public static implicit operator Azure.ResourceManager.AlertsManagement.Models.AlertsSummaryGroupByField (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.AlertsManagement.Models.AlertsSummaryGroupByField left, Azure.ResourceManager.AlertsManagement.Models.AlertsSummaryGroupByField right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct AlertState : System.IEquatable<Azure.ResourceManager.AlertsManagement.Models.AlertState>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public AlertState(string value) { throw null; }
+        public static Azure.ResourceManager.AlertsManagement.Models.AlertState Acknowledged { get { throw null; } }
+        public static Azure.ResourceManager.AlertsManagement.Models.AlertState Closed { get { throw null; } }
+        public static Azure.ResourceManager.AlertsManagement.Models.AlertState New { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.AlertsManagement.Models.AlertState other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.AlertsManagement.Models.AlertState left, Azure.ResourceManager.AlertsManagement.Models.AlertState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.AlertsManagement.Models.AlertState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.AlertsManagement.Models.AlertState left, Azure.ResourceManager.AlertsManagement.Models.AlertState right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class DailyRecurrence : Azure.ResourceManager.AlertsManagement.Models.AlertProcessingRuleRecurrence
+    {
+        public DailyRecurrence() { }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct DaysOfWeek : System.IEquatable<Azure.ResourceManager.AlertsManagement.Models.DaysOfWeek>
```

Comment:
`AlertsManagementDayOfWeek` is weird

Reply:
The preferred solution would be to use a common type and in this case, it's the DayOfWeek defined in System. Otherwise, we need to rename the class to avoid collision with other RPs.